product_price_category/views/product_template.xml

```diff
@@ -0,0 +1,23 @@
+<odoo>
+    <!-- Adds price_category_id in form view -->
+    <record id="product_template_form_view" model="ir.ui.view">
+        <field name="model">product.template</field>
+        <field name="inherit_id" ref="product.product_template_form_view" />
+        <field name="arch" type="xml">
+            <field name="list_price" position="after">
+                <field name="price_category_id" />
```

Comment:
Small comment here,
When there is no price category, the field is not visible :+1: 
![image](https://user-images.githubusercontent.com/1043935/211087901-0c328ac6-5ee1-4957-956f-f089dc8a1981.png)

Any chance to add a label?